### ISSUE TITLE
Nit: orphan letter "m" in istioctl-experimental-metrics documentation

### ISSUE THREAD

**Issue by esnible** [istio/istio.io#3392]
Weird orphan letter `m` in https://preliminary.istio.io/docs/reference/commands/istioctl/#istioctl-experimental-metrics

I think the “m” is the command alias. See  _istioctl/cmd/istioctl/metrics.go_ 

Similar https://preliminary.istio.io/docs/reference/commands/istioctl/#istioctl-proxy-config and weird orphan "pc".  Also "ps" on proxy-status.

**Comment by ayj** [comment]
fixed by https://github.com/istio/istio.io/pull/3430

**Comment by ayj** [comment]
`m` and `ps` are command aliases. These are generated by [pkg/collateral/control.go#L188-L192](https://github.com/istio/istio/blob/1686e3ccc6a98ce5f926bbc47a4262db1e4e1ad2/pkg/collateral/control.go#L188-L192.)

```golang
if len(cmd.Aliases) > 0 {
	g.emit("<p>")
	g.emit(cmd.Aliases...)
	g.emit("<p>")
}
```

@geeknoid @esnible - the snippet above should probably include some text about what the values are (e.g. see below). Any thoughts on formatting?


```html
<p>
Command aliases: m
<p>
```

**Comment by esnible** [comment]
Put the aliases in **bold** or _italics_ or `code` to indicate their specialness.

**Comment by geeknoid** [comment]
What I'd like to do is repeat the example command-line that's produced but using the aliased names instead of the full name. This makes it clear what the use case is. Let me see how hard that would be to incorporate in the collateral package.

**Comment by geeknoid** [comment]
Fixed by #3497
